src/main/resources/templates/authentication/register.html

```diff
@@ -19,7 +19,10 @@
 
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
-<head th:insert='parts/head.html :: head'></head>
+<head th:insert='parts/head.html :: head'>
```

Comment:
please move imports of scripts to after </body> before/after th:replace='parts/scripts :: scripts'